### PR TITLE
Fixes #275(Partly): Create Settings to connect SUSI WebChat to SUSI Hardware

### LINKED ISSUE #275: Create Settings to connect SUSI WebChat to SUSI Hardware
**Actual Behaviour**

No setting to connect to SUSI Hardware

**Expected Behaviour**

Create Setting Page for connecting to SUSI Hardware for PR https://github.com/fossasia/susi_hardware/pull/30

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #275 

Changes: 
- Added Dialog for entering Port Hardware

Changes pending
- Adding functionality to test with the Hardware
This also fixes https://github.com/fossasia/susi_hardware/pull/30

Demo Link: 
http://susi.surge.sh

Screenshots for the change: 
screenshot from 2017-06-18 22 36 27

### CODE REVIEW

File: src/components/ChatApp/HardwareComponent.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		// Handle Hardware Logic
+	}
+
+	handleChange(event) {
+        let state = this.state;
+        let serverUrl;
+        if (event.target.name === 'serverUrl'){
+        	serverUrl = event.target.value;
+        	let validServerUrl =
+/(http|ftp|https):\/\/[\w-]+(\.[\w-]+)+([\w.,@?^=%&amp;:~+#-]*[\w@?^=%&amp;~+#-])?/i
```
Comment (betterclever) [inline src/components/ChatApp/HardwareComponent.js]: websocket url begins with 'ws://'. Please change regex.

Comment (rishiraj824) [inline src/components/ChatApp/HardwareComponent.js]: Okay @betterclever

**Comment by rishiraj824** [comment]
@betterclever Made the changes

**Comment by betterclever** [comment]
@rishiraj824, only 'ws' :sweat_smile:

**Comment by rishiraj824** [comment]
@betterclever there you go

**Comment by betterclever** [comment]
Thanks. :smile:

**Pull request merged**